main.cpp

```diff
@@ -13,75 +13,487 @@ What you create in this project will be used as the basis of Project 5 in the co
+    bool hasMixBusCompressor = true;
+    //Amount of money 
+    int cost = 70000;
+    //Turn faders to 0
+    void faders0();
+    //Mix bus compression
+    bool mixBusOn();
+    //Initiate recording
+    void recordFromDesk();
+};
+    /*
+    Thing 6)
+    5 properties:Imac
+        1)Amount of Storage in GB (int)
+        2)Number of pixels in display (int)
+        3)Amount of GPU power in GHz (float)
+        4)Amount of RAM (int)
+        5)Model Number name (std::string)
+    3 things it can do:
+        1)Record inputs to the mixing desk 
+        2)Store the recordings
+        3)Take pictures
+    */
+struct Imac
+{
+    //Amount of Storage in GB 
+    int storageInGB = 256;
+    //Number of pixels in display
+    int pixels = 4056;
+    //Amount of GPU power in GHz
+    int gpuInGHz = 8;
+    //Amount of RAM 
+    int ramInGB = 2;
+    //Model Number name
+    std::string modelName = "TY34550B";
+    //Record inputs in to the mixing desk 
+    void record();
+    //Store the recordings
+    void save();
+    //Take pictures
+    void takePicture();
+};
+    /*
+    Thing 7)
+    5 properties:Analog Compressors
+        1)Number of knobs of parameters 
+        2)Brand and model (std::string)
+        3)Type of analog compressor (std::string)
+        4)Amount of money to buy (float)
+        5)Number of Kilowatts per Hour used (float)
+    3 things it can do:
+        1)Compress incoming audio
+        2)Turn on
+        3)Turn off
+    */
+struct AnalogCompressor
+{
+    //Number of knobs of parameters
+    int numberOfKnobs = 4;
+    //Brand and model
+    std::string brand = "Drawmer";
+    //Type of analog compressor
+    std::string type = "FET";
+    //Amount of money to buy
+    float price = 1999.99f;
+    //Number of Kilowatts per Hour used
+    int wattsPerHour = 200;
+    //Compress incoming audio
+    void compress (int ratio, float threshold);
+    //Turn on
+    void turnOn();
+    //Turn off
+    void turnOff();
+};
+    /*
+    Thing 8)
+    5 properties:Analog Reverb
+        1)Number of presets (int)
+        2)Number of largest size parameter (float)
+        3)Number of knobs (int)
+        4)Amount of money to buy (float)
+        5)colour (std::string)
+    3 things it can do:
+        1)Apply bright plate reverbs
+        2)Apply dark plate reverbs
+        3)Apply large room reverb
+    */
+struct AnalogReverb
+{
+    //Number of presets
+    int numOfpreSets = 200;
+    //)Number of largest size parameter
+    float numOfMaxSizeVerb = 1000;
+    //Number of knobs
+    int numOfKnobs = 12;
+        struct Parameters
+        {
+            float roomSize = 100.f;
+            int decayInMiliSeconds = 10;
+            int presetNum = 1;
+            float delayInMiliSeconds = 100;
+            bool lowCutFilter = true;
+
+            void type (std::string type1 = "Plate", std::string type2 = "Spring");
+            void accessParameters();
+            void resetParameters();
+
+
+        };    
```

Comment:
Fix this indentation
This nested class should be lined up with the other member variables
```
struct Foo
{
    int memberVar = 2;
    struct Nested
    {
        int nestedMember = 3;
    };
};
```